src/views/GameListView.jsx

```diff
@@ -79,8 +79,21 @@ const Content = styled.div`
 
 const GameListView = () => {
   const { t } = useTranslation();
+  const NULL_SEARCH_TERM = "";
 
   const [gameList, setGameList] = React.useState([]);
+  const [searchTerm, setSearchTerm] = React.useState(NULL_SEARCH_TERM);
+
+  const handleChange = (event) => {
+    setSearchTerm(event.target.value);
+  };
+
+  const filteredGameList = React.useMemo(() => {
```

Comment:
@jrmi Probably something still missing here: I checked when this filter was called, and it's triggered on any `searchTerm` change, even if this value was already used before. Searching for "e", then "ee", then "e" triggers 3 calls to this function, when I was expecting only 2.

Reply:
This is not  parametized memoization, it only keep the last value as long as dependencies not updated, so this is normal!